relayer/relays/beefy/polkadot-listener.go

```diff
@@ -82,36 +82,36 @@ func (li *PolkadotListener) scanCommitments(
 			validatorSetID := result.SignedCommitment.Commitment.ValidatorSetID
 			nextValidatorSetID := uint64(result.MMRProof.Leaf.BeefyNextAuthoritySet.ID)
 
+			if validatorSetID < currentValidatorSet || validatorSetID > currentValidatorSet+1 {
+				return fmt.Errorf("commitment has unexpected validatorSetID: blockNumber=%v validatorSetID=%v expectedValidatorSetID=%v",
+					committedBeefyBlock,
+					validatorSetID,
+					currentValidatorSet,
+				)
```

Comment:
It might be clearer to phrase this as the exact 2 cases we expect:
```suggestion
			if validatorSetID != currentValidatorSet && validatorSetID != currentValidatorSet+1 {
				return fmt.Errorf("commitment has unexpected validatorSetID: blockNumber=%v validatorSetID=%v currentValidatorSetID=%v",
					committedBeefyBlock,
					validatorSetID,
					currentValidatorSet,
				)
```